Makefile

```
@@ -87,10 +84,19 @@ composer-build:
 	composer install --no-dev --prefer-dist
 
 # Installs npm dependencies
-.PHONY: npm
-npm:
+.PHONY: npm-install
+npm-install:
+ifeq (,$(wildcard $(CURDIR)/package.json))
+	cd js && $(npm) npm run dev-install
```

Do not append `npm` as parameter.
```suggestion
	cd js && $(npm) run dev-install
```

---

🙈